pom.xml

```diff
@@ -46,7 +46,7 @@
     <parent>
         <groupId>ddf</groupId>
         <artifactId>ddf-parent</artifactId>
-        <version>1.0.5</version>
+        <version>1.0.6-SNAPSHOT</version>
```

Comment:
the maven deploy plugin was failing because of version mismatches
https://github.com/codice/ddf-parent/pull/11

Reply:
might be best to leave this at snapshot for the time being, so we can test the deploy job and iterate before we release a new version of ddf-parent